### ISSUE TITLE
full_name people field

### ISSUE THREAD

**Issue by sreynen** [opensupporter/osdi-docs#261]
Working on a client-server interaction with OSDI. The client, by design, takes name as a single string of text. Maybe OSDI should support this?

**Comment by j-ro** [comment]
I'm not so sure...I think maybe splitting by heuristics is better?

**Comment by joshco** [comment]
Agree with splitting it at the client.  The entity in the best position to know which components are which (first, last, middle) would be the system which collected it from the user.  
As a simple example, a form label that says either 

* Name (first middle last) [............................]
* Name (last, first, middle) [...........................]

Then that knowledge is at the client which presented the form.   
On the other hand, if you inject the full string into OSDI, and then it gets passed along to other systems, that knowledge is lost. So the heuristics of separation should happen as close to the user input as possible.

**Comment by schuyler1d** [comment]
I agree for server-to-server communication, but what about client-to-server communication? What about for loading in data where the split is unknown?

**Comment by j-ro** [comment]
I think the idea is you do the split client side if you don't control the server, or if you do you don't really need OSDI to send in data but your server does the split and presents that over the API later.

As for unknown, I think you apply heuristics to guess.

**Comment by sonyabea** [comment]
closed by committee 9/18/2018.
